src/core/Akka.API.Tests/verify/CoreAPISpec.ApproveCore.DotNet.verified.txt

```diff
@@ -94,6 +94,8 @@ namespace Akka.Actor
         public void CheckReceiveTimeout(bool reschedule = True) { }
         protected void ClearActor(Akka.Actor.ActorBase actor) { }
         protected void ClearActorCell() { }
+        protected virtual Akka.Actor.ActorStarted CreateActorStartedEvent() { }
```

Comment:
Part of the public API but only gets access by components that use custom `ActorCell`s, which are rare